src/templates/admin/announcement.html

```diff
@@ -0,0 +1,44 @@
+{% import "components/css.html" as css %}
+{% extends "base.html" %}
+{% block title%}Public Documents{% endblock %}
+{% block style %}
+  {{ css.css(c, 'base') }}
+{% endblock %}
+{% block content %}
+
+<h1>CubingUSA Announcements</h1>
+
+<div style="margin-top:30px;">
```

Comment:
Is this kind of inline CSS okay, or should I make a separate style file for the announcements?

Reply:
I prefer to not have inline styling. As the project grows it will only get harder to move the inline styling out into separate files.